core/src/main/java/org/infinispan/interceptors/distribution/VersionedDistributionInterceptor.java

```diff
@@ -34,7 +34,7 @@ protected void prepareOnAffectedNodes(TxInvocationContext<?> ctx, PrepareCommand
       // Perform the RPC
       try {
          Map<Address, Response> resps = rpcManager.invokeRemotely(recipients, command, rpcManager.getDefaultRpcOptions(true, DeliverOrder.NONE));
```

Comment:
No. If the prepare is synchronous then I want to throw an OutdatedTopologyException when a node is suspected or a cache isn't running (`checkTxCommandResponses(resps, command)` below), and to retry the command later.